common/dependabot-common.gemspec

```diff
@@ -19,7 +19,7 @@ Gem::Specification.new do |spec|
   spec.files        = []
 
   spec.required_ruby_version = ">= 2.7.0"
-  spec.required_rubygems_version = ">= 2.7.3"
+  spec.required_rubygems_version = ">= 3.3.22"
```

Comment:
In the upcoming #5447 bump to Ruby 3.1 [we have to go to at least `3.3.3`](https://github.com/dependabot/dependabot-core/pull/5447#discussion_r936301618)... I'm fine holding it back to that, but does it save us much holding back on `3.3.3` vs going straight to `3.3.22`?

Reply:
Well, same argument holds for Ruby itself. We don't need to require all users to upgrade to Ruby 3.1 in order to start using it ourselves. Or course if we hold off bumping the minimum Ruby requirement, we can't yet start using new Ruby features, but I don't think that's the main motivation for us to upgrade Ruby?

Reply:
Sounds good. You perfectly captured my thinking... particularly around support right now. Besides, folks who tend to use Dependabot will tend to care about upgrading their dep chains to latest anyway. 😉

Reply:
You ended up right here @deivid-rodriguez , we got bit by this in a few of our internal repos needing to suddenly upgrade their system RubyGems instead of just using what ships by default with Ruby.

So I opened https://github.com/dependabot/dependabot-core/pull/6010 to downgrade the pin to `>= 3.3.7` which matches what comes with Ruby 3.1.2... IMO it's a reasonable ask to say "please stay updated with Ruby" but if they do that, then we promise not to make them futz around further...